From: Rafael Cunha de Almeida <rafael@kontesti.me>
Last-Update: 2011-01-17
Description: Split README file into changelog and README
 Upstream files README and changelog were split to better fit how things
 work in Debian.

diff -Nurd html2ps-1.0b7.old/changelog html2ps-1.0b7.new/changelog
--- html2ps-1.0b7.old/changelog	1969-12-31 21:00:00.000000000 -0300
+++ html2ps-1.0b7.new/changelog	2011-01-17 21:21:26.000000000 -0200
@@ -0,0 +1,25 @@
+Brief history:
+
+  941212 0.1   - First public release - text-only documents.
+  950407 0.2   - Inline images.
+  970609 1.0b  - Support for most of HTML 3.2, configuration files introduced.
+  970809 1.0b1 - Bug fixes. Some HTML 4.0 support, e.g. OBJECT, INS, DEL,
+                 and ACRONYM.
+  000727 1.0b2 - Mostly bug fixes. A title page can be generated. The symbol
+                 set is extended to include information from META elements.
+                 Installation script modified for Windows support.
+  000815 1.0b3 - Bug fixes. Some support for Server Side Includes. Form
+                 element contents can be rendered. Support for netscape
+                 cookies.
+  050225 1.0b4   Various bug fixes.
+  051211 1.0b5   Bug fixes. Added -M option.
+  091021 1.0b6   "Security" fix. Changed behavior for SSI, to avoid having
+                 arbitrary readable files disclosed on a web server that uses
+                 html2ps in a web application (cgi script etc), allowing HTML
+                 files to be uploaded for conversion. Setting up a service like
+                 this is begging for trouble in my opinion. (Is /bin/cat also
+                 a vulnerability, since one can use it in a web app, allowing
+                 uploading arbitrary parameters to it?)
+  100507 1.0b7   Fixed a bug introduced by the previous bug fix...
+Jan Kärrman
+jan@it.uu.se
diff -Nurd html2ps-1.0b7.old/README html2ps-1.0b7.new/README
--- html2ps-1.0b7.old/README	2011-01-17 21:20:46.000000000 -0200
+++ html2ps-1.0b7.new/README	2011-01-17 21:21:39.000000000 -0200
@@ -14,29 +14,5 @@
   contrib/xhtml2ps - A directory containing a GUI frontend for html2ps.
   COPYING          - GNU General Public License.
   README           - This file.
-
-Brief history:
-
-  941212 0.1   - First public release - text-only documents.
-  950407 0.2   - Inline images.
-  970609 1.0b  - Support for most of HTML 3.2, configuration files introduced.
-  970809 1.0b1 - Bug fixes. Some HTML 4.0 support, e.g. OBJECT, INS, DEL,
-                 and ACRONYM.
-  000727 1.0b2 - Mostly bug fixes. A title page can be generated. The symbol
-                 set is extended to include information from META elements.
-                 Installation script modified for Windows support.
-  000815 1.0b3 - Bug fixes. Some support for Server Side Includes. Form
-                 element contents can be rendered. Support for netscape
-                 cookies.
-  050225 1.0b4   Various bug fixes.
-  051211 1.0b5   Bug fixes. Added -M option.
-  091021 1.0b6   "Security" fix. Changed behavior for SSI, to avoid having
-                 arbitrary readable files disclosed on a web server that uses
-                 html2ps in a web application (cgi script etc), allowing HTML
-                 files to be uploaded for conversion. Setting up a service like
-                 this is begging for trouble in my opinion. (Is /bin/cat also
-                 a vulnerability, since one can use it in a web app, allowing
-                 uploading arbitrary parameters to it?)
-  100507 1.0b7   Fixed a bug introduced by the previous bug fix...
 Jan Kärrman
 jan@it.uu.se
